src/channel-slider/channel-slider.js

```diff
@@ -178,7 +178,13 @@ export default class ChannelSlider extends NudeElement {
 		defaultValue: {
 			type: Number,
 			default () {
-				return this.defaultColor.get(this.channel);
+				try {
+					return this.defaultColor.get(this.channel);
+				}
+				catch {
+					let channel = Object.keys(this.space.coords)[0];
+					return this.defaultColor.get(channel);
```

Comment:
If we're just trying to get the first coord, it’s easier to do `this.defaultColor.coords[0]`.

Reply:
Oh, yes. Much easier.  Thanks!